biojava-structure/src/main/java/org/biojava/nbio/structure/SubstructureIdentifier.java

```diff
@@ -115,7 +125,11 @@ public SubstructureIdentifier(String pdbId, List<ResidueRange> ranges) {
 		if(ranges == null) {
```

Comment:
Should be overridden with a PDBId version

Reply:
OK